consumer/src/mode/decoded/mod.rs

```diff
@@ -1,6 +1,9 @@
 pub mod atom;
 pub mod deposited;
+pub mod deposited_curve;
 pub mod fees_transfered;
 pub mod redeemed;
+pub mod redeemed_curve;
+pub mod share_price_changed_curve;
```

Comment:
Also need to add `share_price_changed` for base vault!  This event already is being emitted in the contract.